docs/languages/javascript.md

```diff
@@ -213,6 +213,58 @@ Organize imports can also be automatically when you save a JavaScript file by se
     "source.organizeImports": true
 }
 ```
+## Code Actions on Save
+
+The  `editor.codeActionsOnSave` setting lets you configure a set of Code Actions that are run when a file is saved. For example, you can enable organize imports on save by setting:
+```json
+// On save, run both fixAll and organizeImports source actions
+"editor.codeActionsOnSave": {
+    "source.fixAll": true,
+    "source.organizeImports": true,
+}
+```
+You can also enable or disable which Code Actions are run on save per language using a language specific setting:
+```json
+"[javascript]": {
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+    }
+}
+```
+Here are some source actions:
+
+* `"organizeImports"` -  Enables organize imports on save.
+* `"fixAll"` - Auto Fix on Save computes all possible fixes in one round (for all providers including ESLint).
+* `"fixAll.eslint"` -  Auto Fix only for ESLint.
+* `"addMissingImports"` - Adds all missing imports on save.
+
+The TSLint extension already uses `source.fixAll` Code Action kind to implement fix all and auto fix on save.
+
+You can also set `editor.codeActionsOnSave` to an array of Code Actions to execute in order. You can use this to guarantee that a specific Code Action is always run before or after another one that may conflict with it.
```

Comment:
If you remove some parts make sure (if possible) to keep this one where it is explained the reason why one would prefer to choose the array vs. the object syntax (to execute in order) I think this is important to understand...